tests/conditional_processing/tests.py

```diff
@@ -188,7 +189,34 @@ def test_single_condition_head(self):
         response = self.client.head('/condition/')
         self.assertNotModified(response)
 
+    def test_unquoted(self):
+        """
+        The same quoted ETag should be set on the header regardless of whether
+        etag_func() in condition() returns a quoted or an unquoted ETag.
+        """
+        response_quoted = self.client.get('/condition/etag/')
+        response_unquoted = self.client.get('/condition/unquoted_etag/')
+        self.assertEqual(response_quoted['ETag'], response_unquoted['ETag'])
+
+    # Since it's possible that the matching algorithm could use the wrong value
```

Comment:
I don't think this makes sense as a docstring. It's not explaining what the function does (or tests), it's more of a meta-comment explaining why this test exists at all (as opposed to having no additional tests and relying on `test_unquoted()`, or duplicating all of the tests for both quoted and unquoted values).